app/javascript/react/screens/App/Overview/components/Migrations/InProgressCard.js

```diff
@@ -4,7 +4,7 @@ import { Card, Grid, noop } from 'patternfly-react';
 
 const InProgressCard = ({ title, children, onClick, ...props }) => (
   <Grid.Col sm={12} md={6} lg={4}>
-    <Card matchHeight onClick={onClick} {...props}>
+    <Card id={`${title}-progress-card`} matchHeight onClick={onClick} {...props}>
```

Comment:
I see your screenshot, but for some reason, my browser does this

<img width="837" alt="devtools_-_localhost_8080_migration" src="https://user-images.githubusercontent.com/15141412/43416605-c4778186-9406-11e8-85cc-88d11e85fefb.png">

Reply:
I suppose it would help I pushed my work 😆 This will fix yah